mappings/net/minecraft/entity/ai/control/LookControl.mapping

```diff
@@ -1,16 +1,24 @@
 CLASS net/minecraft/class_1775 net/minecraft/entity/ai/control/LookControl
+	FIELD field_7577 entity Lnet/minecraft/class_1753;
 	FIELD field_7578 yawSpeed F
 	FIELD field_7579 pitchSpeed F
 	FIELD field_7580 active Z
 	FIELD field_7581 lookX D
 	FIELD field_7582 lookY D
 	FIELD field_7583 lookZ D
+	METHOD <init> (Lnet/minecraft/class_1753;)V
+		ARG 1 entity
 	METHOD method_7335 lookAt (DDDFF)V
 		ARG 1 x
 		ARG 3 y
 		ARG 5 z
 		ARG 7 yawSpeed
 		ARG 8 pitchSpeed
+	METHOD method_7336 clampAndWrapAngle (FFF)F
+	METHOD method_7337 (Lnet/minecraft/class_1745;FF)V
+		ARG 1 entity
+		ARG 2 yawSpeed
```

Comment:
since when did rotation values have speed? i think you mean velocity

Reply:
took it from nbt